istioctl/cmd/istioctl/inject.go

```diff
@@ -275,6 +278,8 @@ func init() {
 
 	injectCmd.PersistentFlags().StringVar(&hub, "hub", version.Info.DockerHub, "Docker hub")
 	injectCmd.PersistentFlags().StringVar(&tag, "tag", version.Info.Version, "Docker tag")
+	injectCmd.PersistentFlags().StringVar(&alpineHub, "alpineHub", "docker.io", "The base image docker hub")
```

Comment:
I'm not sure hardcoding 'alpine' is a good idea. 

We now have ISTIO_PROXY_IMAGE env variable allowing the image name to be customized.